src/lib.rs

```diff
@@ -34,6 +113,28 @@ pub struct Command {
 }
 
 impl Command {
+    /// Create a new [`Command`], with the following default configuration:
+    ///
+    /// * Inherit handles of the calling process.
+    /// * Inherit the current drive and directory of the calling process.
+    ///
+    /// Builder methods are provided to change these defaults and otherwise
+    /// configure the process.
+    ///
+    /// # Examples
+    ///
+    /// ```ignore
```

Comment:
There is a `no_run` or `build_only` expressly for that. `ignore` is bad for maintaining documentation

Reply:
I wasn't able to test all of these example at that time. I have another issue with the `Command` import since the crate is using `CreateProcessW` but i want to demonstrate `create_process_w`

Need investigations